rclpy/rclpy/node.py

```diff
@@ -126,6 +137,15 @@ def destroy_client(self, client):
                 return True
         return False
 
+    def destroy_timer(self, timer):
+        for tmr in self.timers:
+            if tmr.timer_handle == timer.timer_handle:
+                _rclpy.rclpy_destroy_entity(
+                    'timer', tmr.timer_handle)
```

Comment:
Should the return value of these C-calls be checked? (That would apply to many other calls as well.)